db/migrate/20230126123008_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] Kindly create an `author_id` and a corresponding index `t.references :author, foreign_key: { to_table: 'users' }`.
- [ ] You may also create post_id and a corresponding index. `t.references :post, foreign_key: true`